src/screens/application/container/applicationContainer.js

```diff
@@ -255,6 +291,7 @@ const mapStateToProps = state => ({
   currentAccountUsername: state.account.currentAccount.name,
   otherAccounts: state.account.otherAccounts,
   pinCode: state.account.pin,
+  currentAccountAuthType: state.account.currentAccount.local,
```

Comment:
lets get just crrentAccount from state because its calling twice time for name and also local thing.